Tasks/Chyzh/Task_3/task3_pt1.py

```diff
@@ -0,0 +1,26 @@
+s = input("Enter text: ")
+while len(s) <= 3:
+    s = input("Enter string consist of different letters more than 3 symbols length: ")
+N = int(input("Enter the number of symbol you want to check repeated: "))
+if (isinstance(N, int) == False) or N > len(s):
```

Comment:
not a good way to perform the check actually, if user inputs some non-digital value it will not be converted to int anyway (the program will stop working), and if the conversion is successful - the N is totally int , nothing to check here